proto/query.proto

```diff
@@ -989,6 +992,7 @@ enum SchemaTableType {
   VIEWS = 0;
   TABLES = 1;
   ALL = 2;
+  UDF_AGGREGATE = 3;
```

Comment:
Do we allow only aggregate UDFs? The vtgate flag is just called `--enable_udfs`. Release notes also say "any UDFs".